lib/index.ts

```diff
@@ -67,9 +72,20 @@ export const storyblokInit = (pluginOptions: SbSDKOptions = {}) => {
   return result;
 };
 
-export const loadStoryblokBridge = () => {
-  return loadBridge(bridgeLatest);
-};
+export const renderRichText = (text: Richtext) : string => {
+  if (text as any === '') {
+    return ""
+  }
+  else if(!text) {
+    console.warn(`${text} is not a valid Richtext object. This might be because the value of the richtext field is empty.
+    
+  For more info about the richtext object check https://github.com/storyblok/storyblok-js#rendering-rich-text`)
+    return ""; 
+  }
+  return resolver.render(text)
+}
+
+export const loadStoryblokBridge = () => loadBridge(bridgeLatest)
```

Comment:
what would be the purpose/idea of this function?

Reply:
@fgiuliani That function was already added a couple of months ago when I started working with Astro. Alex, you and I realized that for some frameworks, there should be the possibility to load the Bridge only, completely decoupled from initializing the API client.

Reply:
@manuelschroederdev totally forgot about that 😅 I was mainly asking why are we creating this kind of "alias".